### DISCUSSION TITLE
Multiple Backupjobs with different timings?

### DISCUSSION THREAD

**Discussion by pcace** [borgbase/vorta#1687] [FAQ]
Hi, 

i dont quite understand the meaning of having two different presets. does it mean that i can for example backup Folder A to repository A every one Hour, and Folder B to repository B every day? 

I have setup vorta like so, but it seems like it only backups one preset. Can someone clarify if something like this would work and if i understood the meaning of Presets correct? 

Thank you so much!!

**Comment by m3nu** [comment]
By preset you mean profile? Yes, each can have their own repo, source folders and timing.

If only one backup runs, maybe the other doesn't have an automatic schedule set?

**Reply by real-yfprojects** [reply]
Before automatic backups are run for a profile you have to run a manual one. Is that the issue here?

**Comment by pcace** [comment]
Hi, thanks for your reply! But - just as a suggestion - couldnt the interface be a little bit clearer? I feel like the "Backup Started" button should somehow indiate what profile is run in the moment? 
<img width="639" alt="Bildschirmfoto 2023-04-09 um 08 38 41" src="https://user-images.githubusercontent.com/1011875/230758468-0c1b0776-b690-467a-94f4-eeee0e55bae4.png">


for me there is no indication if in those two images what job is running (or is there?)
I think the issue really is, that Vorta never ran the "External" (in my example pictures) Profile. but how would i start it if there is already one running?

<img width="1308" alt="Bildschirmfoto 2023-04-09 um 08 37 30" src="https://user-images.githubusercontent.com/1011875/230758499-943a88d1-98f4-4160-b77d-f3d5b5f7e4c7.png">
<img width="1308" alt="Bildschirmfoto 2023-04-09 um 08 37 35" src="https://user-images.githubusercontent.com/1011875/230758501-552bf225-4f44-4cba-bc0b-2fa05e627e67.png">

Cheers and thanks for your help!

**Reply by real-yfprojects** [reply]
> Hi, thanks for your reply! But - just as a suggestion - couldnt the interface be a little bit clearer? I feel like the "Backup Started" button should somehow indiate what profile is run in the moment?

Yes, this should be fixed in the latest version.

> I think the issue really is, that Vorta never ran the "External" (in my example pictures) Profile. 
but how would i start it if there is already one running?

Good question this is not possible at the moment but I seek to change that as part of #1677.
